hr_linkedin_recruitment/__manifest__.py

```diff
@@ -38,7 +38,7 @@
     ],
     'external_dependencies':
         {
-        'python': ['mechanize', 'linkedin'],
+        'python': ['mechanize', 'linkedin', 'MechanicalSoup'],
```

Comment:
I believe this has to be in lowercase: `'python': ['mechanize', 'linkedin', 'mechanicalsoup'],`.